samples/common_samples_prerequisites.md

```diff
@@ -0,0 +1,22 @@
+# Prerequisite Steps For The Samples
+
+## Create a Project in the Google Cloud Platform Console
+
+If you haven't already created a project, create one now. Projects enable you to
+manage all Google Cloud Platform resources for your app, including deployment,
+access control, billing, and services.
+
+1. Open the [Cloud Platform Console][cloud-console].
+1. In the drop-down menu at the top, select **Create a project**.
+1. Give your project a name.
+1. Make a note of the project ID, which might be different from the project
+   name. The project ID is used in commands and in configurations.
+
+## Install the Snapshot Debugger CLI and enable Firebase
+
+Follow the instructions beginning at [Before you
+begin](../../../README.md#before-you-begin) through to and including [Enable
+Firebase for your Google Cloud
+Project](../../../README.md#enable-firebase-for-your-google-cloud-project) to
```

Comment:
I don't think that these links are correct.  Just one directory down?

Reply:
Thanks!